physionet-django/static/sample/create.json

```diff
@@ -0,0 +1,23 @@
+{
+  "op_trainings": {
+    "version": "float",
+    "contents": [
+      {
+        "body": "string",
+        "order": "integer"
+      }
+    ],
+    "quizzes": [
+      {
+        "question": "string",
```

Comment:
Should we support different types of questions, e.g. multiple_choice, short_answer, etc? Maybe not necessary?

Reply:
Not sure how we would grade them if it's short answer. Maybe a better option is to allow for multiple_choice or checkbox (where the user can check many answers rather than just select a single one). This would give you a way to allow for different types of answering questions without introducing a short answer type that we don't know how to handle.

Reply:
Oh yes, my bad earlier i meant to say we can support the checkbox one where there can be more than 1 right answer.

Good point about short answers, we would need to worry about grading them.